crates/fluvio-version-manager/src/common/settings.rs

```diff
@@ -0,0 +1,66 @@
+use std::fs::write;
+use std::path::PathBuf;
+
+use color_eyre::eyre::{Error, Result};
+use serde::{Deserialize, Serialize};
+use semver::Version;
+
+use fluvio_hub_util::fvm::Channel;
+
+use super::workdir::fvm_workdir_path;
+
+pub const SETTINGS_TOML_FILENAME: &str = "settings.toml";
+
+/// The `settings.toml` is in charge of keeping track of the active version
+/// through the default key, which holds the name of the directory under
+/// `~/.fvm/pkgset/default/versions` for the desired default version.
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Settings {
+    /// The active `channel` for the Fluvio Installation
+    pub channel: Option<Channel>,
+    /// The specific version in use
+    pub version: Option<Version>,
+}
+
+impl Settings {
+    /// Used to create an empty `settings.toml` file. This is used when the user
+    /// installs FVM but no version is set yet.
+    pub fn init() -> Result<Self> {
```

Comment:
pub(crate) since this will not be exposed.
rename to `load` which would be more obvious?

Reply:
`load` would mean reading from file no? `init` similar to for example `git init` is more related to creation of a workspace or files.

I would expect `load` to fail if `settings.toml` doesn't exists